app/src/main/res/xml/preferences.xml

```diff
@@ -38,17 +38,6 @@
 
     </PreferenceCategory>
 
-    <PreferenceCategory
-        android:title="@string/preference_category_location">
-
-        <SwitchPreference
-            android:key="allowGps"
```

Comment:
What do you think of the idea of leaving some trace of the key string "allowGps" so that future developers will not accidentally reuse it? (e.g. "// key 'allowGps' was used before - do not reuse it unless you revive the same feature.")

Reply:
Sounds like a good plan, but where should we mention this?

Reply:
I was thinking we could do that in preferences.xml in place of the removed code. Or maybe somewhere in the wiki pages?

Reply:
Makes sense to me - added a comment there. :) Are we all good to merge?